raster-foundry/raster-foundry
# Set upload limit to 10 GiB instead of 1GB

## Overview

This pr sets the upload creation limit to 10 GiB instead of 1GB (supposed to be 10GB, oops) because it's more consistent with the sizes reported by file systems. It will probably make limit usage more of a nuisance, but for reference, 1 GiB = 2 ^ 30 bytes (it's also expressed that way in the Scope.scala file for easier future reference).

### Checklist


## Testing Instructions

- check how to calculate GiB
- `man du` or `du --help` to confirm default size reporting

Closes raster-foundry/annotate#705
